# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e/libdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- e/src/bin/e_start_main.c_orig	2007-05-05 08:55:52.000000000 +0200
+++ e/src/bin/e_start_main.c	2007-05-05 08:56:38.000000000 +0200
@@ -381,7 +381,7 @@
 	func = dlsym(lib, "eet_init");
 	if (!func) goto done;
 	/* precache SHOULD work */
-	snprintf(buf, sizeof(buf), "%s/lib/enlightenment/preload/e_precache.so", _prefix_path);
+	snprintf(buf, sizeof(buf), "%s/enlightenment/preload/e_precache.so", PACKAGE_LIB_DIR);
 	env_set("LD_PRELOAD", buf);
 	printf("E - PRECACHE GOING NOW...\n");
 	fflush(stdout);
